DEN/donations.yml

```diff
@@ -51,3 +51,29 @@
   files:
     - DEN/pchf4.pdf
   announcement_url: https://denadapool.io/donations.html
+  
+- ticker: DEN
+  recipient: 
+    name: Perth Childrens Hospital Foundation
+    address: PO Box 8249, Subiaco East, WA, 6008, Australia
+    tel: "+62 (08) 6456 2222"
+    contact: info@pchf.org.au
+  ada: 34
+  usd: 41.85
+  timestamp: 1627210800
+  files:
+    - DEN/pchf5.pdf
+  announcement_url: https://denadapool.io/donations.html
+  
+  - ticker: DEN
```

Comment:
I think this is the only error in the YAML file, but just double check it with [YAML](https://codebeautify.org/yaml-validator/) validator to be sure.

You may also need to update your repo with the main branch to make sure it is not out of date. Instructions are in the [README.md](https://github.com/CardanoMDP/Donations#keeping-your-repository-up-to-date-with-the-upstream-repository-cardanomdpdonations)